admin-console/src/main/webapp/src/components/CreateEditModal.tsx

```diff
@@ -72,9 +70,10 @@ export const CreateEditModal: React.FunctionComponent<CreateEditModalProps> = (p
                             name="description"
                         /></FormGroup>
                     <ActionGroup>
-                        <Button variant='primary' type='submit'
-                            { ...props.isLoading ? <Spinner /> : props.eventTypeQuery }
-                            onClick={ onSubmitLocal }>{ props.isEdit ? 'Update' : 'Submit' }</Button>
+                        <Button variant='primary' type='submit' isLoading={ props.isLoading } isDisabled={ props.isLoading }
```

Comment:
Using isLoading instead fo the disable and isLoading of the button (didn't know the button had an isLoading and I'm not actually sure if it does what I think it does :D )